commit 8feab1f620d219a71f488110de1b88b8410ecad8
Author: Steven Rowe <sarowe@apache.org>
Date:   Fri Jun 3 15:28:35 2011 +0000

    added comment describing turning off buffering of '.'s printed while you wait
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1131074 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/dev-tools/scripts/poll-mirrors.pl b/dev-tools/scripts/poll-mirrors.pl
index ed9d75a..5a0305b 100755
--- a/dev-tools/scripts/poll-mirrors.pl
+++ b/dev-tools/scripts/poll-mirrors.pl
@@ -47,7 +47,7 @@ unless (defined($version) && $version =~ /\d+(?:\.\d+)+/) {
 }
 
 my $previously_selected = select STDOUT;
-$| = 1;
+$| = 1; # turn off buffering of STDOUT, so "."s are printed immediately
 select $previously_selected;
 
 my $apache_backup_url = "http://www.apache.org/dist//lucene/java/$version/lucene-$version.tgz.asc";
